src/Card/README.md

```diff
@@ -58,9 +59,10 @@ You use `isClickable` prop to add additional `hover` and `focus` styling to the
 () => {
   const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
 
-  return(
+  return (
     <Card style={{ width: isExtraSmall ? "100%" : "18rem" }} isClickable>
-      <Card.ImageCap 
+      <Card.ImageCap
+        skeletonHeight={140}
```

Comment:
If this `Card` isn't in a loading state (no `isLoading` prop), do we need the `skeletonHeight` prop here?